examples/metrics/grafana-dashboards/strimzi-cruise-control.json

```diff
@@ -0,0 +1,2062 @@
+      "targets": [
+        {
+          "expr": "kafka_cruisecontrol_loadmonitor_valid_windows_value{namespace=\"$kubernetes_namespace\",strimzi_io_cluster=\"$strimzi_cluster_name\"}",
+          "format": "time_series",
+          "hide": false,
+          "intervalFactor": 1,
+          "refId": "A"
+        }
+      ],
+      "thresholds": "2",
+      "title": "Valid Windows",
+      "type": "singlestat",
+      "valueFontSize": "200%",
+      "valueMaps": [
+        {
+          "op": "=",
+          "text": "N/A",
+          "value": "null"
+        }
+      ],
+      "valueName": "current"
+    },
+    {
+      "cacheTimeout": null,
+      "colorBackground": false,
+      "colorValue": true,
+      "colors": [
+        "#299c46",
+        "rgba(237, 129, 40, 0.89)",
+        "#d44a3a"
+      ],
+      "datasource": "${DS_PROMETHEUS}",
+      "description": "Number of ongoing executions running for proposals or rebalance",
+      "format": "none",
+      "gauge": {
+        "maxValue": 100,
+        "minValue": 0,
+        "show": false,
+        "thresholdLabels": false,
+        "thresholdMarkers": true
+      },
+      "gridPos": {
+        "h": 4,
+        "w": 4,
+        "x": 8,
+        "y": 0
+      },
+      "id": 38,
+      "interval": null,
+      "links": [],
+      "mappingType": 1,
+      "mappingTypes": [
+        {
+          "name": "value to text",
+          "value": 1
+        },
+        {
+          "name": "range to text",
+          "value": 2
+        }
+      ],
+      "maxDataPoints": 100,
+      "nullPointMode": "connected",
+      "nullText": null,
+      "options": {},
+      "postfix": "",
+      "postfixFontSize": "50%",
+      "prefix": "",
+      "prefixFontSize": "50%",
+      "rangeMaps": [
+        {
+          "from": "null",
+          "text": "N/A",
+          "to": "null"
+        }
+      ],
+      "sparkline": {
+        "fillColor": "rgba(31, 118, 189, 0.18)",
+        "full": false,
+        "lineColor": "rgb(31, 120, 193)",
+        "show": false
+      },
+      "tableColumn": "",
+      "targets": [
+        {
+          "expr": "kafka_cruisecontrol_executor_ongoing_execution_non_kafka_assigner_value{namespace=\"$kubernetes_namespace\",strimzi_io_cluster=\"$strimzi_cluster_name\"}",
+          "format": "time_series",
+          "hide": false,
+          "intervalFactor": 1,
+          "refId": "A"
+        }
+      ],
+      "thresholds": "2",
+      "title": "Ongoing Execution",
+      "type": "singlestat",
+      "valueFontSize": "200%",
+      "valueMaps": [
+        {
+          "op": "=",
+          "text": "N/A",
+          "value": "null"
+        }
+      ],
+      "valueName": "current"
+    },
+    {
+      "cacheTimeout": null,
+      "colorBackground": false,
+      "colorValue": true,
+      "colors": [
+        "#299c46",
+        "rgba(237, 129, 40, 0.89)",
+        "#d44a3a"
+      ],
+      "datasource": "${DS_PROMETHEUS}",
+      "description": "Balancedness Score",
```

Comment:
Can we get some better descriptions? For example I have no idea what does _Balancedness Score_, which value is good, which bad etc. ... I guess 100 is bad here since it is orange in the screenshot?

Reply:
Yes, I reverted the logic in the dashboard, now higher values are green while lower are orange and then red. I used 50 (for red) and 80 (orange) as thresholds.

Reply:
So is 100 a maximum? TBH, I think the Cruise Control is relative niche ... so the descriptions are IMHO quite important since many people will be looking at it as I'm ... without having a clue.

BTW: Should we have a sample alert for this as well?

Reply:
Yes 100 is the max.
Regarding the alerts I would add them in a different PR as already did for the bridge metrics.

Reply:
I think it is useful to have this metric as a one-stop-shop for quickly checking the state of the cluster. To be clear this is the current balancedness score of the cluster as calculated by the anomaly detector (using `KafkaCruiseControlUtils.balancednessCostByGoal` CC method) and isupdated every 5 mins by default (can be changed by setting `anomaly.detection.interval.ms`). I wonder if we need some docs to describe how this is calculated and what it means?

Reply:
So can we add this stuff to the description?